Pokefans/Web.config

```diff
@@ -23,7 +23,8 @@
         <add key="FromEmailName" value="Pokefans.NET" />
         <add key="VariableLessPath" value="G:\Projekte\Pokefans\pokeboot\less\variables.less" />
         <add key="ContentStylesheetPath" value="/var/www/pokefans/static/stylesheets/content/" />
-        <add key="Domain" value="pokefans.localhost" />
+        <add key="Domain" value="pokefans.rocks" />
+      <add key="Port" value="50000" />
```

Comment:
N.B. we should include a not-in-repo config file for stuff like this, because probably everybody will change this as fits.
Also we need a second value for custom HTTPS-Ports.